apistructs/project_pipeline.go

```diff
@@ -14,14 +14,29 @@
 
 package apistructs
 
+import "path/filepath"
+
 type PipelineCategory string
 
+type ProjectPipelineType string
+
 const (
-	CategoryBuildDeploy   = "build-deploy"
-	CategoryBuildArtifact = "build-artifact"
-	CategoryOthers        = "others"
+	CategoryBuildDeploy                       = "build-deploy"
+	CategoryBuildArtifact                     = "build-artifact"
+	CategoryOthers                            = "others"
+	CICDPipelineType      ProjectPipelineType = "cicd"
+
+	SourceType = "erda"
```

Comment:
separate a single code block, should not put together with PipelineCategory

Reply:
and `SourceType` -> `SourceTypeErda`

Reply:
and `CICDPipelineType` -> `PipelineTypeCICD`

Reply:
done